tensorflow_addons/losses/python/focal_loss.py

```diff
@@ -0,0 +1,144 @@
+# Copyright 2015 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Implements Focal loss"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import abc
+import six
+import numpy as np
+
+import tensorflow as tf
+import tensorflow.keras.backend as K
+from tensorflow.python.keras import losses
```

Comment:
The `tensorflow.python.*` is internal to TF and shouldn't be used to access anything. You can use `tf.keras.losses` instead.